### PR TITLE
Add recipe for `smudge`

### PR DESCRIPTION
**Pull request by jkdufair**
### Brief summary of what the package does

Control the Spotify application from within Emacs

### Direct link to the package repository

https://github.com/danielfm/smudge

### Your association with the package

Contributor. Have worked closely with @danielfm to prepare the package for Melpa

### Relevant communications with the upstream package maintainer

https://github.com/danielfm/smudge/issues/44

### Checklist

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] `M-x checkdoc` is happy with my docstrings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [ ] I have confirmed some of these without doing them

- [x] Also confirmed working with gccemacs

### CODE REVIEW

File: recipes/smudge [hunk 1]
```diff
@@ -0,0 +1,5 @@
+(smudge
+	:fetcher github
+	:repo "danielfm/smudge"
+	:branch "master"
+	:files ("*.el"))
```
Comment (riscy) [inline recipes/smudge:5]: Default recipe should work here (and gives you room to expand), and we only require you to specify `:branch` if you aren't using your repo's default branch -- and in any case MELPA prefers to use the default branch when possible.
```suggestion
	:repo "danielfm/smudge")
```

**Review by riscy — COMMENTED** [review]
Thanks for this, and thank you for your close adherence to the checklist.

**Comment by jkdufair** [comment]
Applied! Thanks @riscy

**Comment by riscy** [comment]
Great thanks!  Welcome both to MELPA. :)

**Pull request merged**